src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/ReportedAdjudicationControllerTest.kt

```diff
@@ -159,7 +159,7 @@ class ReportedAdjudicationControllerTest : TestControllerBase() {
       getMyAdjudications().andExpect(status().isOk)
       verify(reportedAdjudicationService).getMyReportedAdjudications(
         "MDI",
-        LocalDate.now(), LocalDate.now(), Optional.empty(),
+        LocalDate.now().minusDays(3), LocalDate.now(), Optional.empty(),
```

Comment:
We need a default-dates test for all completed adjudications as well. I am not sure one exists at all yet.